src/PaymentGateways/Gateways/Stripe/StripePaymentElementGateway/stripePaymentElementGateway.tsx

```diff
@@ -89,6 +89,9 @@ interface StripeGateway extends Gateway {
     settings?: StripeSettings;
 }
 
+/**
+ * @since 3.0.0
```

Comment:
I think we need an unreleased tag here as well to document the changes related to the address fields.

Reply:
resolved d6a35b0e